project/AppDependencies.scala

```diff
@@ -22,8 +22,8 @@ object AppDependencies {
 
```

Comment:
can you change the wiremock dependency to "com.github.tomakehurst" % "wiremock-jre8" % "2.27.2" % scope and remove the overrides from here + build.sbt